src/lib.rs

```diff
@@ -1,9 +1,8 @@
 ///! An API client for the [Cloudflare API](https://api.cloudflare.com)
 extern crate chrono;
 extern crate reqwest;
-extern crate serde;
 #[macro_use]
-extern crate serde_derive;
```

Comment:
This is interesting - is there a reason this is no longer needed?

Reply:
Serde now has a feature which re-exports the `derive` macros from this crate. I figured it was cleaner to just depend on the serde crate rather than both serde and serde-derive.